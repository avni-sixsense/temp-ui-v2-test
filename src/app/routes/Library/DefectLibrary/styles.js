import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  paperBg: {
    backgroundColor: '#FFFFFF'
  },
  root: {
    width: '100%'
  },
  paper: {
    width: '100%',
    marginBottom: theme.spacing(2)
  },
  table: {
    minWidth: 750,
    '& thead > tr > th': {
      paddingBottom: '8px !important'
    },
    '& tbody > tr > td': {
      borderBottom: 'none'
    }
  },
  tableHeadBorder: {
    minWidth: 750,
    '& thead > tr > th': {
      paddingBottom: '8px',
      borderBottom: 'none'
    },
    '& tbody > tr > td': {
      borderBottom: 'none'
    }
  },
  visuallyHidden: {
    border: 0,
    clip: 'rect(0 0 0 0)',
    height: 1,
    margin: -1,
    overflow: 'hidden',
    padding: 0,
    position: 'absolute',
    top: 20,
    width: 1
  },
  divider: {
    width: '100%',
    height: '48px',
    display: 'flex',
    justifyContent: 'center',
    flexDirection: 'column',
    alignItems: 'center'
  },
  dot: {
    width: '6px',
    height: '6px',
    marginLeft: '0.5px',
    background: '#B3C7CE',
    borderRadius: '5px'
  },
  dash: {
    height: '20px',
    borderLeft: '0.5px dashed #000000'
  },
  solid: {
    height: '20px',
    borderLeft: 'none'
  },
  tableSection: {
    marginTop: theme.spacing(4)
  }
}));

export default useStyles;
