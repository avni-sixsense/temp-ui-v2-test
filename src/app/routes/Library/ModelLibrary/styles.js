import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  paperBg: {
    backgroundColor: '#FFFFFF'
  },
  root: {
    width: '100%'
  },
  paper: {
    width: '100%',
    marginBottom: theme.spacing(2)
  },
  tableSection: {
    marginTop: theme.spacing(4)
  },
  table: {
    minWidth: 750,
    '& thead > tr > th': {
      paddingBottom: '8px'
    },
    '& tbody > tr > td': {
      borderBottom: 'none'
    }
  },
  visuallyHidden: {
    border: 0,
    clip: 'rect(0 0 0 0)',
    height: 1,
    margin: -1,
    overflow: 'hidden',
    padding: 0,
    position: 'absolute',
    top: 20,
    width: 1
  },
  divider: {
    width: '100%',
    height: '48px',
    display: 'flex',
    justifyContent: 'center',
    flexDirection: 'column',
    alignItems: 'center'
  },
  dot: {
    width: '6px',
    height: '6px',
    marginLeft: '0.5px',
    background: '#B3C7CE',
    borderRadius: '5px'
  },
  dash: {
    height: '20px',
    borderLeft: '0.5px dashed #000000'
  },
  solid: {
    height: '20px',
    borderLeft: 'none'
  },
  tableContainer: {
    overflow: 'hidden',
    maxHeight: 'calc(100vh - 200px)',
    '& [class*="MuiTableContainer-root"]': {
      overflow: 'auto',
      maxHeight: 'calc(100vh - 360px)',
      // maxHeight: '1500px',
      '&::-webkit-scrollbar': {
        width: '8px'
      },
      '&::-webkit-scrollbar-track': {
        boxShadow: 'inset 0 0 0px white',
        borderRadius: '5px'
      },
      '&::-webkit-scrollbar-thumb': {
        backgroundColor: ' #dfdcdc',
        borderRadius: '10px'
      },
      '&::-webkit-scrollbar-thumb:hover': {
        backgroundColor: '#cecece'
      }
    }
  }
}));

export default useStyles;
