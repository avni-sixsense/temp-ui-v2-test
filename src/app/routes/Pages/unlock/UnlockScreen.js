import Button from '@material-ui/core/Button';
import Grid from '@material-ui/core/Grid';
import Paper from '@material-ui/core/Paper';
import { makeStyles } from '@material-ui/core/styles';
import { alpha } from '@material-ui/core/styles/colorManipulator';
import Typography from '@material-ui/core/Typography';
import CustomInput from 'app/components/Base/Input';
import useForm from 'app/hooks/useForm';
import Close from 'assests/images/close.svg';
import React from 'react';

const useStyles = makeStyles(theme => ({
  paper: {
    width: '630px',
    display: 'flex',
    padding: 0,
    flexDirection: 'column',
    borderRadius: '10px',
    border: `1px solid ${alpha(theme.colors.marineBlue, 0.13)}`,
    boxShadow: `0 10px 20px 5px ${alpha(theme.colors.azure, 0.1)}`,
    marginTop: theme.spacing(5),
    marginBottom: theme.spacing(2),
    // minHeight: '530px',
    marginLeft: 'auto',
    marginRight: 'auto'
  },
  unlockForm: {
    width: '100%', // Fix IE 11 issue.
    padding: theme.spacing(4)
  },
  title: {
    ...theme.typeFace.futura125,
    fontWeight: 500,
    lineHeight: '2rem',
    textAlign: 'center',
    marginTop: theme.spacing(3)
  },
  closeBtn: {
    lineHeight: '2rem',
    textAlign: 'left',
    marginTop: theme.spacing(3)
  }
}));

const initialState = {
  firstName: '',
  lastName: '',
  email: '',
  phoneNo: '',
  message: ''
};

function UnlockScreen(props) {
  const classes = useStyles();

  const { state, dispatch } = useForm(initialState);
  const { firstName, lastName, email, phoneNo, message } = state;

  const onChange = e => {
    dispatch({
      type: 'form',
      field: e.target.name,
      value: e.target.value
    });
  };

  const onSubmit = () => {};

  return (
    <>
      <Paper elevation={3} className={classes.paper}>
        <Grid container justifyContent='center' spacing={5}>
          <Grid item xs={10} sm={10}>
            <Typography className={classes.title} variant='h1'>
              Unlock Backend
            </Typography>
          </Grid>
          <Grid item xs={1} sm={1}>
            <img src={Close} alt='' className={classes.closeBtn} />
          </Grid>
        </Grid>

        <form className={classes.unlockForm} noValidate>
          <Grid container justifyContent='center' spacing={4}>
            <Grid item xs={12} sm={6}>
              <CustomInput
                value={firstName}
                onChange={onChange}
                name='firstName'
                label='First Name'
                required
              />
            </Grid>
            <Grid item xs={12} sm={6}>
              <CustomInput
                value={lastName}
                onChange={onChange}
                name='lastName'
                label='Last Name'
                required
              />
            </Grid>
            <Grid item xs={12} sm={6}>
              <CustomInput
                value={email}
                onChange={onChange}
                name='email'
                label='Email Address'
                required
              />
            </Grid>
            <Grid item xs={12} sm={6}>
              <CustomInput
                value={phoneNo}
                onChange={onChange}
                name='phoneNo'
                label='Phone Number'
                required
              />
            </Grid>
            <Grid item xs={12} sm={12}>
              <CustomInput
                value={message}
                onChange={onChange}
                name='message'
                label='Message'
                required
                multiline
              />
            </Grid>
            <Grid container item justifyContent='center'>
              <Button id='unlockBtn' onClick={() => onSubmit()}>
                Request For Unlock
              </Button>
            </Grid>
          </Grid>
        </form>
      </Paper>
    </>
  );
}

export default UnlockScreen;
