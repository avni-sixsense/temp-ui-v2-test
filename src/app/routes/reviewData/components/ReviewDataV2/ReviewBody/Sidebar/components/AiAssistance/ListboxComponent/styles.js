import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  inputBase: {
    borderRadius: '8px',
    width: '100% !important',
    backgroundColor: '#162236',
    color: '#FFFFFF',
    '& input': {
      color: '#FFFFFF',
      '&::placeholder': {
        color: '#31456A',
        opacity: 1
      },
      '&:-ms-input-placeholder': {
        color: '#31456A'
      },
      '&::-ms-input-placeholder': {
        color: '#31456A'
      }
    },
    '& .MuiAutocomplete-tag': {
      backgroundColor: '#7C3AED',
      color: '#FFFFFF',
      fontSize: '0.8125rem'
    },
    '& .MuiOutlinedInput-notchedOutline': {
      borderRadius: '8px'
    }
  },
  lightInputBase: {
    borderRadius: '8px',
    width: '100% !important',
    backgroundColor: theme.colors.grey[2],
    color: theme.colors.grey[21],
    '& input': {
      color: theme.colors.grey[21],
      '&::placeholder': {
        color: theme.colors.grey[10],
        opacity: 1
      },
      '&:-ms-input-placeholder': {
        color: theme.colors.grey[10]
      },
      '&::-ms-input-placeholder': {
        color: theme.colors.grey[10]
      }
    },
    '& .MuiAutocomplete-tag': {
      backgroundColor: theme.colors.grey[0],
      color: theme.colors.grey[19],
      fontSize: '0.8125rem'
    },
    '& .MuiOutlinedInput-notchedOutline': {
      borderRadius: '8px'
    },
    '& :hover': {
      '& [class*="MuiOutlinedInput-notchedOutline-"]': {
        borderColor: theme.colors.blue[600],
        borderWidth: '1px',
        boxShadow: '0px 0px 10px rgba(37, 99, 235, 0.1), 0px 0px 1px #2563EB'
      }
    },
    '& .Mui-focused': {
      '& [class*="MuiOutlinedInput-notchedOutline-"]': {
        borderColor: theme.colors.blue[600],
        borderWidth: '1px',
        boxShadow: '0px 0px 10px rgba(37, 99, 235, 0.1), 0px 0px 1px #2563EB'
      }
    }
  },
  paper: {
    width: '300px',
    wordWrap: 'break-word',
    maxWidth: '300px',
    boxShadow: 'none',
    backgroundColor: '#1C2A42',
    color: '#FFFFFF',
    '& p': {
      color: '#FFFFFF',
      fontSize: '0.875rem !important'
    },
    '& li': {
      '& p': {
        msWordBreak: 'break-all',
        overflowX: 'auto'
      },
      '& div': {
        fontSize: '0.875rem',
        color: '#FFFFFF',
        backgroundColor: '#1C2A42'
      },
      paddingLeft: 0
    },
    '& ul': {
      overflowY: 'scroll',
      // maxHeight: '200px',
      '&::-webkit-scrollbar ': {
        width: 3
      },

      /* Track */
      '&::-webkit-scrollbar-track': {
        borderRadius: 10
      },

      /* Handle */
      '&::-webkit-scrollbar-thumb': {
        background: '#31456A',
        borderRadius: 10
      },

      /* Handle on hover */
      '&::-webkit-scrollbar-thumb:hover': {
        background: '#EEEEEE'
      }
    }
    // maxHeight: 190,
    // overflowY: 'scroll',
    // '&::-webkit-scrollbar ': {
    // 	width: 3,
    // },

    // /* Track */
    // '&::-webkit-scrollbar-track': {
    // 	borderRadius: 10,
    // },

    // /* Handle */
    // '&::-webkit-scrollbar-thumb': {
    // 	background: '#31456A',
    // 	borderRadius: 10,
    // },

    // /* Handle on hover */
    // '&::-webkit-scrollbar-thumb:hover': {
    // 	background: '#EEEEEE',
    // },
  },
  lightPaper: {
    width: '300px',
    wordWrap: 'break-word',
    maxWidth: '300px',
    boxShadow: 'none',
    backgroundColor: theme.colors.grey[0],
    color: theme.colors.grey[19],
    '& p': {
      color: theme.colors.grey[19],
      fontSize: '0.875rem !important'
    },
    '& li': {
      '& p': {
        msWordBreak: 'break-all',
        overflowX: 'auto'
      },
      '& div': {
        fontSize: '0.875rem',
        color: theme.colors.grey[19],
        backgroundColor: theme.colors.grey[0]
      },
      paddingLeft: 0
    },
    '& ul': {
      overflowY: 'scroll',
      // maxHeight: '200px',
      '&::-webkit-scrollbar ': {
        width: 3
      },

      /* Track */
      '&::-webkit-scrollbar-track': {
        borderRadius: 10
      },

      /* Handle */
      '&::-webkit-scrollbar-thumb': {
        background: '#31456A',
        borderRadius: 10
      },

      /* Handle on hover */
      '&::-webkit-scrollbar-thumb:hover': {
        background: theme.colors.grey[4]
      }
    }
    // maxHeight: 190,
    // overflowY: 'scroll',
    // '&::-webkit-scrollbar ': {
    // 	width: 3,
    // },

    // /* Track */
    // '&::-webkit-scrollbar-track': {
    // 	borderRadius: 10,
    // },

    // /* Handle */
    // '&::-webkit-scrollbar-thumb': {
    // 	background: theme.colors.grey[0],
    // 	borderRadius: 10,
    // },

    // /* Handle on hover */
    // '&::-webkit-scrollbar-thumb:hover': {
    // 	background: theme.colors.grey[4],
    // },
  },
  option: {
    '&[aria-selected="true"]': {
      backgroundColor: theme.colors.grey[17]
    },
    '&[data-focus="true"]': {
      backgroundColor: theme.colors.grey[16]
    },
    '& div': {
      backgroundColor: 'transparent !important'
    }
  },
  lightOption: {
    '&[aria-selected="true"]': {
      backgroundColor: theme.colors.grey[2]
    },
    '&[data-focus="true"]': {
      backgroundColor: theme.colors.grey[1]
    },
    '& div': {
      backgroundColor: 'transparent !important'
    }
  },
  popperDisablePortal: {
    position: 'relative',
    width: 'auto !important',
    // maxHeight: 184,
    overflowY: 'auto',
    backgroundColor: '#1C2A42',
    color: '#FFFFFF',
    '&::-webkit-scrollbar ': {
      width: 3
    },

    /* Track */
    '&::-webkit-scrollbar-track': {
      borderRadius: 10
    },

    /* Handle */
    '&::-webkit-scrollbar-thumb': {
      background: '#31456A',
      borderRadius: 10
    },

    /* Handle on hover */
    '&::-webkit-scrollbar-thumb:hover': {
      background: '#EEEEEE'
    }
  },
  lightPopperDisablePortal: {
    position: 'relative',
    width: 'auto !important',
    // maxHeight: 184,
    overflowY: 'auto',
    backgroundColor: theme.colors.grey[0],
    color: theme.colors.grey[19],
    '&::-webkit-scrollbar ': {
      width: 3
    },

    /* Track */
    '&::-webkit-scrollbar-track': {
      borderRadius: 10
    },

    /* Handle */
    '&::-webkit-scrollbar-thumb': {
      background: theme.colors.grey[0],
      borderRadius: 10
    },

    /* Handle on hover */
    '&::-webkit-scrollbar-thumb:hover': {
      background: theme.colors.grey[4]
    }
  },
  popperDisablePortalDefect: {
    position: 'relative',
    width: '300px !important'
  },
  item: {
    display: 'flex',
    alignItems: 'center',
    cursor: 'pointer',
    backgroundColor: theme.colors.grey[14],
    padding: theme.spacing(0.25, 0.75, 0.5, 0.75),
    borderRadius: '2px',
    boxShadow: theme.colors.shadow['base'],
    '& p': {
      color: theme.colors.grey[0],
      fontWeight: 600,
      fontSize: '0.875rem',
      marginRight: theme.spacing(0.6875)
    },
    '& svg': {
      color: theme.colors.grey[8],
      fontWeight: 900,
      fontSize: '0.625rem'
    },
    '& ul': {
      padding: '0px !important'
    }
  },
  lightItem: {
    display: 'flex',
    alignItems: 'center',
    cursor: 'pointer',
    border: `0.8px solid ${theme.colors.grey[4]}`,
    backgroundColor: theme.colors.grey[0],
    padding: theme.spacing(0.25, 0.75, 0.5, 0.75),
    borderRadius: '2px',
    boxShadow: theme.colors.shadow['base'],
    '& p': {
      color: theme.colors.grey[19],
      fontWeight: 600,
      fontSize: '0.875rem',
      marginRight: theme.spacing(0.6875)
    },
    '& svg': {
      color: theme.colors.grey[7],
      fontWeight: 900,
      fontSize: '0.625rem'
    },
    '& ul': {
      padding: '0px !important'
    }
  },
  popper: {
    zIndex: 100
  },
  menuPaper: {
    backgroundColor: '#1C2A42',
    padding: theme.spacing(1)
  },
  lightMenuPaper: {
    backgroundColor: theme.colors.grey[0],
    padding: theme.spacing(1),
    border: `0.5px solid ${theme.colors.grey[4]}`
  },
  menuDefect: {
    '& .MuiMenu-paper': {
      boxShadow: '-3px 2px 30px rgba(0, 0, 0, 0.14)',
      borderRadius: '3px',
      border: '1px solid #E8EDF1',
      minWidth: '300px',
      height: '402px',
      '& .MuiMenu-list': {
        paddingLeft: '0 !important',
        paddingRight: '0 !important',
        paddingTop: '0 !important',
        paddingBottom: '0 !important',
        height: '100%'
      },
      '& .MuiAutocomplete-option': {
        padding: 0
      }
    }
  },
  dateMenu: {
    '& .MuiMenu-paper': {
      boxShadow: '-3px 2px 30px rgba(0, 0, 0, 0.14)',
      borderRadius: '3px',
      border: '1px solid #E8EDF1',
      minWidth: '340px',
      minHeight: '356px',
      padding: '8px',
      '& .MuiMenu-list': {
        padding: '0 !important',
        width: '100% !important',
        height: '100%'
      }
    }
  },
  menuItem: {
    position: 'absolute',
    bottom: 0,
    backgroundColor: 'transparent',
    '&:hover': {
      backgroundColor: 'transparent'
    },
    '&:focus': {
      backgroundColor: 'transparent'
    }
  },
  list: {
    height: '250px',
    overflowY: 'scroll'
  },
  search: {
    width: '80%',
    border: 'none',
    borderBottom: '0.5px solid #E3E3E3',
    fontSize: '12px',
    lineHeight: '32px',
    color: '#000000',
    opacity: 0.5
  },
  cancelBtn: {
    borderRadius: '3px',
    fontWeight: 'normal',
    fontSize: '14px',
    lineHeight: 'normal',
    border: '0px',
    color: '#02435D',
    backgroundColor: 'rgba(2, 67, 93, 0.06)',
    padding: '8px 24px',
    '&:hover': {
      backgroundColor: 'rgba(2, 67, 93, 0.12)'
    },
    '&:disabled': {
      background: '#E0E0E0',
      color: '#000D12',
      opacity: 0.24
    }
  },
  applyBtn: {
    borderRadius: '3px',
    fontWeight: 'normal',
    fontSize: '14px',
    lineHeight: 'normal',
    border: '0px',
    color: '#FFFFFF',
    backgroundColor: '#F56C6C !important',
    padding: theme.spacing(1, 3),
    '&:hover': {
      backgroundColor: '#E14242 !important'
    },
    '&:disabled': {
      background: '#E0E0E0 !important',
      color: '#000D12',
      opacity: 0.24
    }
  },
  datePicker: {
    backgroundColor: 'yellow'
  },
  filterCount: {
    backgroundColor: theme.colors.grey[10],
    color: theme.colors.grey[0],
    borderRadius: '1000px',
    paddingLeft: theme.spacing(0.5),
    paddingRight: theme.spacing(0.5),
    marginLeft: theme.spacing(0.5),
    marginRight: theme.spacing(0.25),
    paddingBottom: theme.spacing(0.125)
  },
  lightfilterCount: {
    backgroundColor: theme.colors.grey[2],
    color: theme.colors.grey[19],
    borderRadius: '1000px',
    paddingLeft: theme.spacing(0.5),
    paddingRight: theme.spacing(0.5),
    marginLeft: theme.spacing(0.5),
    marginRight: theme.spacing(0.25),
    paddingBottom: theme.spacing(0.125)
  },

  scroll: {
    '& div': {
      '&::-webkit-scrollbar ': {
        width: 3
      },

      /* Track */
      '&::-webkit-scrollbar-track': {
        borderRadius: 10
      },

      /* Handle */
      '&::-webkit-scrollbar-thumb': {
        background: '#31456A',
        borderRadius: 10
      },

      /* Handle on hover */
      '&::-webkit-scrollbar-thumb:hover': {
        background: '#EEEEEE'
      }
    }
  },
  loadingText: {
    marginBottom: 0,
    paddingLeft: theme.spacing(0.5)
  },
  AIoutputFilter: {
    '& p': {
      color: theme.colors.grey[9],
      fontStyle: 'italic',
      fontWeight: 400,
      fontSize: '0.6875rem'
    },
    '& svg': {
      cursor: 'pointer'
    }
  },
  lightAIoutputFilter: {
    '& p': {
      color: theme.colors.grey[10],
      fontStyle: 'italic',
      fontWeight: 400,
      fontSize: '0.6875rem'
    },
    '& svg': {
      cursor: 'pointer'
    }
  },
  popperFilter: {
    display: 'flex',
    alignItems: 'center',
    padding: theme.spacing(1.5, 1),
    width: '227px !important',
    backgroundColor: theme.colors.grey[16],
    border: `0.4px solid ${theme.colors.grey[12]}`,
    boxShadow: theme.colors.shadow.xl,
    bordeRadius: '6px',
    cursor: 'default'
  },
  lightPopperFilter: {
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'space-between',
    padding: theme.spacing(1.5, 1.5, 1.5, 1),
    width: '227px !important',
    backgroundColor: theme.colors.grey[0],
    border: `0.4px solid ${theme.colors.grey[2]}`,
    boxShadow: theme.colors.shadow.xl,
    bordeRadius: '6px',
    cursor: 'default'
  },
  clearFilterBtn: {
    border: 'none'
  }
}));

export default useStyles;
