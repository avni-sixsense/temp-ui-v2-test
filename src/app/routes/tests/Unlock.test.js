import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
import { fireEvent, render } from '@testing-library/react';
import theme from 'app/configs/theme';
import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import UnlockScreen from '../Pages/unlock/UnlockScreen';

function InputField(props) {
  const { onSubmit } = props;
  return (
    <BrowserRouter>
      <MuiThemeProvider theme={theme}>
        <UnlockScreen />
      </MuiThemeProvider>
    </BrowserRouter>
  );
}

const DataEntry = (container, name, data) => {
  let field;
  if (name !== 'message') {
    field = container.querySelector(`input[name=${name}]`);
  } else {
    field = container.querySelector(`textarea`);
  }
  fireEvent.change(field, { target: { value: data } });
  return field;
};

test('sets the value to the upper version of the value', () => {
  const onSubmit = jest.fn();
  const { container } = render(<InputField />);

  const FirstName = DataEntry(container, 'firstName', 'John');
  const LastName = DataEntry(container, 'lastName', 'Doe');
  const Email = DataEntry(container, 'email', 'newemail@sixsense.ai');
  const Phone = DataEntry(container, 'phoneNo', '1234567890');
  const Message = DataEntry(container, 'message', 'hahaha');

  const UnlockBtn = container.querySelector('button#unlockBtn');

  expect(FirstName.value).toEqual('John');
  expect(LastName.value).toEqual('Doe');

  expect(Email.value).toEqual('newemail@sixsense.ai');
  expect(Phone.value).toEqual('1234567890');

  expect(Message.value).toEqual('hahaha');

  fireEvent.click(UnlockBtn);
  // expect(onSubmit).toHaveBeenCalled()
});
