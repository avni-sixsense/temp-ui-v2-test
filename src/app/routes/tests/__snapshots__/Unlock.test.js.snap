// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test case for <ForgetPassword> page should have same markup 1`] = `
<div
  className="MuiPaper-root makeStyles-paper-1 MuiPaper-elevation3 MuiPaper-rounded"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-5 MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-sm-10"
    >
      <h1
        className="MuiTypography-root makeStyles-title-3 MuiTypography-h1"
      >
        Unlock Backend
      </h1>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 MuiGrid-grid-sm-1"
    >
      <img
        alt=""
        className="makeStyles-closeBtn-4"
        src="close.svg"
      />
    </div>
  </div>
  <form
    className="makeStyles-unlockForm-2"
    noValidate={true}
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4 MuiGrid-justify-xs-center"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root makeStyles-formControl-167"
        >
          <p
            className="MuiFormHelperText-root"
          >
            <span
              className="MuiTypography-root makeStyles-inputLabel-168 MuiTypography-caption"
            >
              First Name
              <span
                style={
                  Object {
                    "color": "red",
                  }
                }
              >
                *
              </span>
            </span>
          </p>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root makeStyles-input-170 MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              id="base-input-"
              name=""
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
              value=""
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-216 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-217"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root makeStyles-formControl-167"
        >
          <p
            className="MuiFormHelperText-root"
          >
            <span
              className="MuiTypography-root makeStyles-inputLabel-168 MuiTypography-caption"
            >
              Last Name
              <span
                style={
                  Object {
                    "color": "red",
                  }
                }
              >
                *
              </span>
            </span>
          </p>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root makeStyles-input-170 MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              id="base-input-"
              name=""
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
              value=""
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-216 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-217"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root makeStyles-formControl-167"
        >
          <p
            className="MuiFormHelperText-root"
          >
            <span
              className="MuiTypography-root makeStyles-inputLabel-168 MuiTypography-caption"
            >
              Email Address
              <span
                style={
                  Object {
                    "color": "red",
                  }
                }
              >
                *
              </span>
            </span>
          </p>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root makeStyles-input-170 MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              id="base-input-"
              name=""
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
              value=""
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-216 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-217"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6"
      >
        <div
          className="MuiFormControl-root makeStyles-formControl-167"
        >
          <p
            className="MuiFormHelperText-root"
          >
            <span
              className="MuiTypography-root makeStyles-inputLabel-168 MuiTypography-caption"
            >
              Phone Number
              <span
                style={
                  Object {
                    "color": "red",
                  }
                }
              >
                *
              </span>
            </span>
          </p>
          <div
            className="MuiInputBase-root MuiOutlinedInput-root makeStyles-input-170 MuiInputBase-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              className="MuiInputBase-input MuiOutlinedInput-input"
              disabled={false}
              id="base-input-"
              name=""
              onAnimationStart={[Function]}
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="text"
              value=""
            />
            <fieldset
              aria-hidden={true}
              className="PrivateNotchedOutline-root-216 MuiOutlinedInput-notchedOutline"
              style={
                Object {
                  "paddingLeft": 8,
                }
              }
            >
              <legend
                className="PrivateNotchedOutline-legend-217"
                style={
                  Object {
                    "width": 0.01,
                  }
                }
              >
                <span
                  dangerouslySetInnerHTML={
                    Object {
                      "__html": "&#8203;",
                    }
                  }
                />
              </legend>
            </fieldset>
          </div>
          
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12"
      >
        <div
          className="MuiFormControl-root MuiFormControl-fullWidth"
        >
          <p
            className="MuiFormHelperText-root"
          >
            <span
              className="MuiTypography-root makeStyles-inputLabel-168 MuiTypography-caption"
            >
              Message
              <span
                style={
                  Object {
                    "color": "red",
                  }
                }
              >
                *
              </span>
            </span>
          </p>
          <div
            className="MuiFormControl-root MuiTextField-root makeStyles-textArea-169 MuiFormControl-fullWidth"
          >
            <div
              className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline"
              onClick={[Function]}
            >
              <textarea
                aria-invalid={false}
                autoFocus={false}
                className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
                defaultValue="type your message here…"
                disabled={false}
                onAnimationStart={[Function]}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                required={false}
                rows={4}
              />
              <fieldset
                aria-hidden={true}
                className="PrivateNotchedOutline-root-216 MuiOutlinedInput-notchedOutline"
                style={
                  Object {
                    "paddingLeft": 8,
                  }
                }
              >
                <legend
                  className="PrivateNotchedOutline-legend-217"
                  style={
                    Object {
                      "width": 0.01,
                    }
                  }
                >
                  <span
                    dangerouslySetInnerHTML={
                      Object {
                        "__html": "&#8203;",
                      }
                    }
                  />
                </legend>
              </fieldset>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-justify-xs-center"
      >
        <button
          className="MuiButtonBase-root MuiButton-root MuiButton-text"
          disabled={false}
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Request For Unlock
          </span>
        </button>
      </div>
    </div>
  </form>
</div>
`;
